libsolidity/formal/SymbolicTypes.cpp

```diff
@@ -426,7 +426,7 @@ smtutil::Expression maxValue(frontend::TypePointer _type)
 	)
 		return TypeProvider::uint(160)->maxValue();
 	if (auto const* enumType = dynamic_cast<EnumType const*>(_type))
-		return enumType->numberOfMembers();
+		return enumType->numberOfMembers() - 1;
```

Comment:
You could probably make a member `maxValue()` for `EnumType`. `EnumType::numberOfMembers() - 1` is used at a couple of other places.

Reply:
That is true. But that's a change in the `ast` module, so maybe somebody more senior than me should propose that :)

Reply:
It's only adding a helper function to `Types.h`. Not really a big deal. Feel free to do it, if you like :)

Reply:
Let's not establish a cargo cult here. Any suggestions to change any code is perfectly fine.